Add new overlay advtools fastcombat - skip combat anims and announcements (#1385)
DFHack/scripts: changelog.txt

- `autocheese`: automatically make cheese using barrels that have accumulated sufficient milk

## New Features

> - `advtools`: new overlay ``advtools.fastcombat``, while active it allows you to instantly skip many combat actions

Reviewer: ```suggestion
- `advtools`: new overlay ``advtools.fastcombat``; allows you to skip combat animations and the announcement "More" button
```
